weechat_otr_test/mock_weechat.py

```diff
@@ -113,14 +115,14 @@ def restore(self):
 
     def snapshot_weechat_dir(self):
         tar_io = io.BytesIO()
-        with tarfile.open(fileobj=tar_io, mode='w') as tar:
+        with closing(tarfile.open(fileobj=tar_io, mode='w')) as tar:
```

Comment:
If we have to pull in another module just to use `with` it seems better just to do the old `tarfile.open` ... `tar.close()`.